packages/css-framework/index.scss

```diff
@@ -1,90 +1,8 @@
-// Globals
-$__colors: null;
-$__zIndex: null;
-$__typography: null;
-$__spacing: null;
-$__shadows: null;
-$__breakpoints: null;
-$__states: null;
-
-@import "src/config";
-
-// Functions
-@import "src/abstracts/functions/map-deep-get";
-@import "src/abstracts/functions/map-get-default";
-@import "src/abstracts/functions/non-destructive-map-merge";
-@import "src/abstracts/functions/color";
-@import "src/abstracts/functions/spacing";
-@import "src/abstracts/functions/spacing";
-@import "src/abstracts/functions/typography";
-
-// Contexts
-@import "src/contexts/light";
-
-// import Variables
-@import "src/vars/breakpoints";
-@import "src/vars/colors";
-@import "src/vars/spacing";
-@import "src/vars/typography";
-@import "src/vars/state";
-@import "src/vars/zindex";
-
-// Mixins
-@import "src/abstracts/mixins/breakpoint";
-@import "src/abstracts/mixins/clearfix";
-@import "src/abstracts/mixins/cover";
-@import "src/abstracts/mixins/hover";
-@import "src/abstracts/mixins/pseudo";
-@import "src/abstracts/mixins/shadows";
-@import "src/abstracts/mixins/truncate";
-@import "src/abstracts/mixins/typography";
-@import "src/abstracts/mixins/zindex";
-
-// Base
-@import "src/base/normalize";
```

Comment:
Agree re: normalize being imported into every instance. Every other file in `library` doesn't output any CSS. `normalize`, `box-sizing`, and `fonts` are the only files in this instance that output CSS, therefore meaning they'll be duplicated with every import. I don't know how best to tackle this unfortunately.

Reply:
It really feels like there should be something in webpack that can handle this.